### PR TITLE
RELATED: SD-122 add unit test generating min/max and tick positions

### PR DESCRIPTION
**Pull request by naibinh**

### CODE REVIEW

**Comment by naibinh** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2327/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1746/ : SUCCESS in 4m 23s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/924/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2328/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/531/ : SUCCESS in 16m 06s

File: src/components/visualizations/chart/highcharts/plugins/zeroAlignYAxis.ts [hunk 1]
```diff
@@ -206,23 +259,21 @@ export function getTickPositioner() {
...
-            axesCanon = [];
 
             for (const axis of chart.axes) {
                 if (axis.coll === 'yAxis') {
                     if (typeof axis.max === 'undefined' || axis.max === null ||
                         typeof axis.min === 'undefined' || axis.min === null) {
                         // Don't have min/max values for both axes yet.
                         // Clean up processed values from other axis.
-                        minmaxLookup = undefined;
+                        // minmaxLookup = undefined;
```
[the remark this answers is not in the document]
Comment (naibinh) [reply src/components/visualizations/chart/highcharts/plugins/zeroAlignYAxis.ts]: I did, thank you.

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2335/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1752/ : SUCCESS in 5m 05s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/496/ : SUCCESS in 26s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1753/ : SUCCESS in 5m 32s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/927/ : UNSTABLE in 37m 05s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/443/ : SUCCESS in 23m 03s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/497/ : SUCCESS in 36s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1758/ : SUCCESS in 5m 30s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/928/ : SUCCESS in 19m 22s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/444/ : FAILURE in 20m 11s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2350/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1765/ : SUCCESS in 4m 24s

**Comment by naibinh** [comment]
I have rebased the code and rebuild the job, it succeeded.
https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/446/console

@pbenes , please help me add label again. Hope everything goes well.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/505/ : SUCCESS in 49s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/1786/ : SUCCESS in 18m 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/943/ : SUCCESS in 23m 27s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/456/ : SUCCESS in 21m 14s

**Pull request merged**